### ISSUE TITLE
On workspace restart: browser tabs don't reopen non-default workspaces

### ISSUE THREAD

**Issue by havogt** [gitpod-io/gitpod#1647]
### Describe the bug
This is probably between bug report and feature request as I couldn't find documentation for my workflow:
I have a project with one main language (python) and a subproject in a different language (C++). For plugins to work nicely I open the subproject in a separate tab (<default_workspace_url>/mysubfolder). This works nicely.
However, if the gitpod workspace is stopped and restarted, both browser tabs open the root workspace and the state (e.g. open files) of the subproject-workspace is lost. (workspace is used for two different things in gitpod? for the gitpod workspace instance and the theia workspace?)

### Steps to reproduce
1. Start a workspace
2. Open a sub-folder of the root workspace in a new browser tab and open some files
3. Stop and restart the gitpod workspace (keeping browser tabs open)
4. Both browser tabs open the root workspace and only the state of the root workspace is persisted 

### Expected behavior
Instead of 4 above, I would expect that each browser tab re-opens with the correct (sub-)workspace and previous state of each workspace.

**Comment by corneliusludmann** [comment]
Indeed, this is not supported yet. Thanks for this feature request.

**Comment by havogt** [comment]
Ping.

(Also, does it make sense to stale a feature request by a bot, after it was recognized as such by a friendly human? The effect on me is changing my perception of the gitpod team from "great community, they instantly reply, even if they cannot work on everything the user wants" to "in the end they don't care, and outsource the dirty work to a bot". ;) But maybe it's the only way to deal with > 1k issues at this point.)
